Packs/AzureCompute_v2/pack_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Azure Compute v2",
+    "description": "Create and Manage Azure Virtual Machines",
+    "support": "XSOAR",
```

Comment:
@yuvalbenshalom what should be the support here?

Reply:
xsoar